main.nf

```diff
@@ -9,279 +9,796 @@
-
-     nf-core/slamseq v${workflow.manifest.version}
-    =======================================================
 
     Usage:
 
     The typical command for running the pipeline is as follows:
 
-    nextflow run nf-core/slamseq --reads '*_R{1,2}.fastq.gz' -profile standard,docker
+    nextflow run nf-core/slamseq --reads '*_R{1,2}.fastq.gz' -profile docker
 
     Mandatory arguments:
-      --reads                       Path to input data (must be surrounded with quotes)
-      --genome                      Name of iGenomes reference
-      -profile                      Configuration profile to use. Can use multiple (comma separated)
-                                    Available: standard, conda, docker, singularity, awsbatch, test
+      --input [file]                  Tab-separated file containing information about the samples in the experiment (see docs/usage.md)
 
-    Options:
-      --singleEnd                   Specifies that the input is single end reads
+      -profile [str]                  Configuration profile to use. Can use multiple (comma separated)
+                                      Available: conda, docker, singularity, test, awsbatch, <institute> and more
 
-    References                      If not specified in the configuration file or you wish to overwrite any of the references.
-      --fasta                       Path to Fasta reference
+    Options:
+      --genome [str]                  Name of iGenomes reference
+
+    References                        If not specified in the configuration file or you wish to overwrite any of the references
+      --fasta [file]                  Path to fasta reference
+      --bed [file]                    Path to 3' UTR counting window reference
+      --mapping [file]                Path to 3' UTR multimapper recovery reference (optional)
+      --vcf [file]                    Path to VCF file for genomic SNPs to mask T>C conversion (optional)
+
+    Processing parameters
+      --trim5 [int]                   Number of basepairs to trim from 5' end of the read
+      --polyA [int]                   Maximum number of As at the 3' end of a read.
+      --multimappers [bool]           Activate multimapper retainment strategy
+      --quantseq [bool]               Deactivate nucleotide-conversion aware scoring
+      --endtoend [bool]               Use a end to end alignment algorithm for mapping.
+      --min_coverage [int]            Minimimum coverage to call a SNP.
+      --var_fraction [float]          Minimimum variant fraction to call a SNP.
+      --conversions [int]             Minimum number of conversions to count a read as converted read
+      --baseQuality [int]             Minimum base quality to filter conversions
+      --readLength [int]              Read length of processed reads
+      --pvalue [float]                P-value cutoff for MA plot
+      --skipTrimming [bool]           Skip trimming step
+      --skipDeseq2 [bool]             Skip trimming step
```

Comment:
I think we do advise to use only lowercase parameters

Reply:
`snake_case_please` ;-)